gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/AttributeCacheKey.java

```diff
@@ -3,6 +3,9 @@
 import java.util.Map;
 import java.util.Objects;
 
+// TODO replace with org.hypertrace.core.grpcutils.context.ContextualKey
```

Comment:
Stopped the scope creep here, but request context has a generated cache key that was added several years ago and can replace this.